packages/bootstrap-chat/README.md

```diff
@@ -0,0 +1,157 @@
+# bootstrap-chat
+
+A set of UI components based on [Bootstrap v3](https://react-bootstrap-v3.netlify.app/components/alerts/) for facilitating chat-based tasks for Mephisto.
+
+## Installation
+
+```bash
+npm install --save bootstrap-chat mephisto-task
+
+# Note that `mephisto-task` is a peer-dependency of this library.
+```
+
+Install from local folder (e.g. for local development):
+
+```bash
+cd packages/bootstrap-chat
+npm link
+
+cd <app folder>
+npm link bootstrap-chat
+
+# If you're getting an invalid hooks error (https://reactjs.org/warnings/invalid-hook-call-warning.html),
+# you can also do the following to ensure that both the app
+# and bootstrap-chat are using the same version of React:
+# 
+# cd packages/bootstrap-chat
+# npm link ../<app folder>/node_modules/react
+
+```
+
+## Usage
+
+This package contains both individual components for building a chat UI, as well as precomposed components that can get you going quickly.
+
+We generally recommend starting with the precomposed components, unless you have an advanced use-case. We provide escape-hatches for easy customization of common behaviors.
+
+### Example 1: The pre-composed ChatApp component
+
+The most simplest way to get going with 0 customizations is just the following:
+
+```jsx
+import { ChatApp } from "bootstrap-chat";
+
+ReactDOM.render(<ChatApp />, document.getElementById("root"));
+```
+
+### Example 2: Modifying ChatApp
+
+From here, you can easily opt into adding custom components and behaviors via render prop overrides.
+
+Let's consider a contrived example where we would like every incoming chat message to appear suffixed with 5 exclamation marks.
+
+```jsx
+
+import { ChatApp, ChatMessage } from "bootstrap-chat";
+
+function MyChatApp() {
+    return (
+        <ChatApp
+            renderMessage={({ message }) => <ChatMessage message={message.text + "!!!!!"} />}
+        />);
+}
+
+```
+
+Here we replace the rendering of the chat message with the `renderMessage` prop. This is one of the provided optional render overrides, for the others see below.
+
+However, note that we also import the `<ChatMessage />` component and use that as part of the rendering process. We don't have to do this portion, and could easily just create our own isolated implementation, e.g. something like `<div>{message.text + "!!!!!"}</div>`. However this example illustrates how we can opt out of default behavior, while still having available to us primitives and core components that can get us back some of that default functionality at our discretion. In this case, we
+ll still have the look and feel of the original chat message component, however with a custom message property.
```

Comment:
nit: newline breaks "we'll". Also may be worthwhile to drop one of the however's in the above block for flow.